superset/assets/cypress_build.sh

```diff
@@ -12,5 +12,5 @@ cd "$(dirname "$0")"
 npm install -g yarn
 yarn
 npm run build
-npm run cypress run
+npm run cypress run --record --parallel --key 1f958c86-be14-44d9-8d08-fad68da06811
```

Comment:
What is `--record` for?

Reply:
`--record` is used for setting up cypress with Dashboard https://dashboard.cypress.io/#/projects/fbf96q/runs

Reply:
Oh.. I see. Thought the name was abt video recording or sth.